COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
Copyright on this repo is implicitly shared between all the authors (if your country adheres to the [Berne convention](https://en.wikipedia.org/wiki/Berne_Convention)). It's correct to update the COPYING file to reflect that. Our mistake is in not doing it sooner.